### ISSUE TITLE
Empreinte carbone du vélo

### ISSUE THREAD

**Issue by publicodes** [datagir/nosgestesclimat#960]
Je suis surpis par votre annonce d'une empreinte carbone exactement nulle pour le vélo.
1-alors que vous intégrez la fabrication d'une voitue, vous n'intégrez pas celle d'un vélo ?
2- lorsque je fais du vélo, je parcours des km et donc produit/dépense de l'énergie et son CO2 associé (respiration).
Petit calcul d'ordre de grandeur (si vous avez le calcul exact, je suis intéressé) : un humain normal produit environ 800 g de CO2 (https://www.futura-sciences.com/planete/questions-reponses/rechauffement-climatique-respiration-humaine-contribue-t-elle-rechauffement-climatique-14867/) pour 2000 kCal par jour. Un trajet quotidien d'une heure à vélo (vitesse modérée) consomme environ 500 kCal (https://www.regivia.com/comment-maigrir-conseils-solutions-trucs-et-astuces/calcul-des-depenses-energetiques-en-calories-par-sports-et-activites/depenses-caloriques-consommees-pour-le-velo-bicyclette-et-cyclisme/#besoin), soit, en considérant que la chimie du corps reste identique, environ 200 g de CO2. A supposer que l'on fasse 10 km pendant cette heure, cela representerait environ 20 g de CO2 par km. C'est 2 fois plus que ce que vous donnez pour le train.
3- donc, si on considère, comme vous le faites, un trajet quotidien moyen de 5 km/jour, on devrait trouver aux environs de 35 kg de CO2 par an.
4- ce n'est pas beaucoup MAIS il ne faudrait peut-être pas laisser l'idée (si elle est fausse) qu'un trajet en vélo, quelque soit sa longueur, ne produit aucun CO2 (comme si cette énergie était gratuite), ni qu'elle pourrait remplacer avantagement tout type de transport (dont le train) pour le même kilométrage produit.
3- il faut sans doute ajouter l'empreinte carbone de la production de l'aliment que le cycliste a consommé.

PS : je viens de trouver ceci : https://www.caradisiac.com/Empreinte-carbone-du-cycliste-57748.htm) qui annonce jusqu'à 1 kg de CO2 par km parcouru à vélo.

Cordialement

Depuis la page : `https://nosgestesclimat.fr/documentation/transport/vélo`

> Ce ticket a été créé automatiquement par notre robot depuis notre [page de contribution](https://nosgestesclimat.fr/contribuer).

**Comment by laem** [comment]
> alors que vous intégrez la fabrication d'une voitue, vous n'intégrez pas celle d'un vélo ?

Oui, c'est une imprécision du calcul, mais en termes de priorité elle est de l'ordre de ~100kg/10 ans, donc 10kg/an, donc de l'ordre de d'une semaine de viande. Dès que l'on trouve une référence, on l'ajoute :) 

> 2 

Les émissions de la respiration ne sont pas à intégrer dans le calcul, car c'est un cycle de carbone certes anthropiques, mais immédiat. Par contre, les émissions de la production de nourriture liée à l'utilisation d'hydrocarbures fossiles, au méthane ou et au changement d'affectation des sols sont des sommes différentes qui doivent être intégrées, comme vous le dites : 

> il faut sans doute ajouter l'empreinte carbone de la production de l'aliment que le cycliste a consommé.



Mais le calcul de ce surplus d'énergie et donc d'émissions est complexe, car : 
- nos sociétés aujourd'hui consomment en moyenne *trop* de calories. Il n'est donc pas du tout évident qu'il y ait un quelconque surplus de consommation pour une utilisation non sportive du vélo. Reste à définir les limites.
- l'activité physique est essentielle au bien être humain, de façon absolue hors des normes de l'époque. Si l'on considère que cette activité (encore une fois, définir les limites ?) relève de la consommation à comptabiliser dans le bilan, alors il faut revoir beaucoup d'autres choses prises pour acquises. Ce qui n'empêche aucunement de travailler le point au-dessus et le comptabiliser le cas échéant (le surplus de calories) et en réduire l'empreinte

> 4 

Je vous propose de faire de notre discussion une note synthétique sur la page `https://nosgestesclimat.fr/documentation/transport/vélo/km` :) 

Merci pour le retour très constructif :)

**Comment by laem** [comment]
Peut-être une piste de réflexion sur le VO2 ici : https://twitter.com/chachauffedur/status/1380045671970041856

**Comment by J-LeJeune** [comment]
Bonjour,

Le rapport du Shift Project "ÉTUDE COMPARATIVE DE L'IMPACT CARBONE DE L'OFFRE DE VÉHICULES" fournit certaines réponses aux questions posées ci-dessus. Le rapport est en accès libre au lien suivant:
https://theshiftproject.org/wp-content/uploads/2020/02/2020-02-04_%C3%89tude-de-limpact-carbone-de-loffre-de-v%C3%A9hicules_V1.pdf

Concernnant les émissions associées au modes de déplacement actifs, l'encadré 4 du rapport fournit une méthode et des ordres de grandeur. Le rapport fournit aussi les émissions liées à la fabrication et la fin de vie d'un vélo (encadré 5).

La conclusion assez contre-intuitive du rapport est que le facteur d'émission d'un "vélo classique" (p. 66, 26 + 7 = 33 gCO2e/vkm) est supérieur à celui d'un vélo à assistance électrique en France (p. 67, 17 + 12 = 29 gCO2e/vkm). La limite à ce raisonnement est l'impact bénéfique de l'effort physique sur la santé et donc les émissions des soins évités, voire la prise en compte de l'allongement de la durée de vie (pas fastoche a priori).

**Comment by laem** [comment]
Voir aussi https://arxiv.org/pdf/2103.04464.pdf
